_maps/atlas.json

```diff
@@ -25,6 +25,21 @@
         "emergency": "emergency_atlas"
     },
     "mine_disable": 1,
-    "ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter",
-	"starmap_path": "config/starmap/starmap.json"
+    "ship_type": "/obj/structure/overmap/nanotrasen/frigate/starter",
```

Comment:
```suggestion
    "ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter",
```
/obj/structure/overmap/nanotrasen/frigate/starter doesn't exist.

Reply:
I made it exist. See the nanotrason.dm changes.

Reply:
I saw that, it's just that the atlas's overmap broke on live, so something might've gone wrong with that.